src/sgame/sg_bot_util.cpp

```diff
@@ -1752,16 +1751,12 @@ float CalcAimPitch( gentity_t *self, glm::vec3 &targetPos, float launchSpeed )
 		check = Square( Square( v ) ) - g * ( g * Square( dr ) + 2 * dz * Square( v ) );
 	}
 
-	//calculate required angle of launch
-	angle1 = atanf( ( Square( v ) + sqrtf( check ) ) / ( g * dr ) );
-	angle2 = atanf( ( Square( v ) - sqrtf( check ) ) / ( g * dr ) );
+	// the same equation with a + sign would also give a correct result,
```

Comment:
Is this true? Are there no edge cases which lead to the previous line?

Reply:
following along with explainations:

```
            sqrt(c)              >             −sqrt(c)                 because sqrt is a positive value
       v² + sqrt(c)              >         v² − sqrt(c)                 we add the same thing on both side
      (v² + sqrt(c))/(g × x)     >        (v² − sqrt(c))/(g × dr)       we divide by a *positive* value on both side
atan( (v² + sqrt(c))/(g × x) )   >  atan( (v² − sqrt(c))/(g × dr) )     atan is strictly monotonous and growing: atan(1)>atan(0)>atan(-0.01) etc
```

Reply:
So the value with the − is always smaller. And if c=0, then this becomes an equality and the result is the same on both sides.

Keeping the information around is practical though, because I might do some tracing to see if the path is actually doable (without obstacles)